temporal-testing/build.gradle

```diff
@@ -1,11 +1,30 @@
+plugins {
+    id 'application'
+}
+
 description = '''Temporal Workflow Java SDK testing'''
 
 dependencies {
+    api group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.30'
```

Comment:
I'm not sure adding this as an `api` dependency was the right move? Exposing `slf4j-api` as an api-scoped dependency makes sense, though.

Reply:
Very little understanding went into this, feel free to change it!